sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumReceiptView.java

```diff
@@ -39,7 +39,7 @@ public EthereumReceiptView(EthereumReceipt receipt, EthereumTransaction tx, BigI
         to = (tx.getTo().isPresent()) ? Numeric.toHexString(tx.getTo().get().address()) : null;
         cumulativeGasUsed = Numeric.encodeQuantity(receipt.consensusDataReceipt().cumulativeGasUsed());
         gasUsed = Numeric.encodeQuantity(receipt.gasUsed());
-        contractAddress = receipt.contractAddress().length != Account.ADDRESS_SIZE ? null : Numeric.toHexString(receipt.contractAddress());
+        contractAddress = receipt.contractAddress().get().length != Account.ADDRESS_SIZE ? null : Numeric.toHexString(receipt.contractAddress().get());
```

Comment:
I don't think we need to check the length anymore. Previously length could zero or ADDRESS_SIZE, but now it will always be ADDRESS_SIZE - or the whole thing is `None`.